python/cuml/tests/test_simpl_set.py

```diff
@@ -108,7 +108,7 @@ def test_simplicial_set_embedding(
     random_state = 42
     metric = "euclidean"
     initial_alpha = 1.0
-    a, b = UMAP.find_ab_params(1.0, 0.1)
```

Comment:
We used to deliberately expose `find_ab_params` as a static method to get `a` and `b` from `min_dist` and `spread`. Doing so would now require `umap-learn` otherwise. This is not very important, but may be handy to some.